buildScript/loadScript.js

```diff
@@ -2,37 +2,25 @@
  * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
  */
 
-
-function loadScript(loader, script, callback){
+(function(loader, script){
     var scriptTag = document.createElement('script');
-    scriptTag.type = 'text/javascript';
+        scriptTag.type = 'text/javascript';
 
-    const url = getScriptURL(loader) + script;
-    if (scriptTag.readyState){  //IE
-        scriptTag.onreadystatechange = function(){
-            if (['loaded', 'complete'].includes(scriptTag.readyState)) {
-                scriptTag.onreadystatechange = null;
-                callback && callback();
+        const getScriptURL = (loader) => {
```

Comment:
I saw this originally bug forgot to comment.  I don't think you should do this a a arrow function.  Since it is not precompiled it should be in ES5.
```
var getScriptURL =  function(loader) {
````